Community-Aktivitäten/README.md

```diff
@@ -0,0 +1,16 @@
+# JavaLand Community-Aktivitäten
+
+Die JavaLand wird von ehrenamtlichen Java-Enthusiasten aus Deutschland, Österreich und der Schweiz organisiert und gestaltet.
+Neben dem Vortragsprogramm sind die Community-Aktivitäten ein integraler Bestandteil und einzigartiges Alleinstellungsmerkmal der Konferenz.
+Im Unterschied zu Vorträgen handelt es sich dabei um kürzere und längere Veranstaltungen, bei denen jeder Teilnehmer selbst *aktiv* wird.
+Dies können beispielsweise Workshops sein, Diskussionsrunden, "Hands-In" Seminare... was auch immer Euch an *aktiven* Formaten einfällt!
```

Comment:
Hast du bewusst „Hands-In“ anstatt Hands-on geschrieben?

Reply:
Nö, Tippfehler.